### DISCUSSION TITLE
Termín(y) příštího tréninku na K a v troji přímo na webu

### DISCUSSION THREAD

**Discussion by Grejdy** [bosancz/bosan.cz#580] [Ideas]
**Jaký problém tvůj nápad řeší?**
Aby se člověk dozvěděl kdy je příští trénink, tak musí sescrollovat v programu až úplně dolu a rozkliknout nenápadný odkaz na formulář. To není úplně přívětivé pro malé (i velké) děti, co by se tam rádi poprvé přihlásili a vůbec netuší, jak na to.

**Popiš možnosti řešení**
Přidat  do programu/aktuálně textík s datem příštích tréninků (klidně do té části o konkrétním tréninku), které se budou aktualizovat podle formuláře/nějaké tabulky k tomu (mám dojem, že nějaký takový rozpis existuje).

**Comment by jhavelik** [comment]
@Grejdy uff zní to hezky nicméně realita je taková, že tréninky jsou někde v Google Sheetu a dostat je odtamtud na web je peklo.
Jako workaround mě napadá po tom co si trenér vezme trénink tak ho i vypsat normálně do programu. Jen bude potřeba se domluvit s Punčochou/Leilou ohledně schvalování akce.

Napadá vás jak to řešit jinak? @SmallhillCZ @DavidTvrdy 

BTW opět spíš téma do discussions než do issues.

**Reply by DavidTvrdy** [reply]
Vypisování termínů do programu by mohlo zvednout povědomí o trénincích a zvýšit účast, na druhou stranu to má nějaký nevýhody.. Narušili bychom tím trochu aktuální systém [vypsat akce, rozebrat akce, schválit akce, zveřejnit akce]. Pořádání tréninků (aspoň teď) je občas takové trochu chaotické (ve smyslu že se občas tréninky ruší, takže by zase bylo potřeba rušit i akce). Pro správce programu to je značná práce navíc, nebo spíš teda museli by reagovat na žádosti o vypsání a schválení akce dost rychle, nevím jak moc dopředu bývají tréninky rozebrané.

Já bych asi zvolil to řešení co popisuje Grejdy, v sekci o tréninku/kanále mít blok s datem příštího naplánovaného tréninku z tabulky. A když jednou budeme mít feed událostí místo sekce Aktuálně, tak aby se tam o tom generovala zpráva.

**Comment by SmallhillCZ** [comment]
U Čárkovníku jsme jako backend použili Google sheet a nakonec to bylo docela v pohodě, tak by to možná zas takový peklo být nemuselo, ale nedokážu říct teď.
